connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -436,6 +441,17 @@ ReplicationPolicy replicationPolicy() {
         return getConfiguredInstance(REPLICATION_POLICY_CLASS, ReplicationPolicy.class);
     }
 
+    @SuppressWarnings("unchecked")
+    ForwardingAdmin getForwardingAdmin(Map<String, Object> config) {
```

Comment:
We tend to not use the `get` prefix for getters.

Reply:
removed the prefix